acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/SmokeTest.java

```diff
@@ -300,136 +299,75 @@ public void testFullBuildFlowWithFreeStyleJobRefAndPRTrigger() throws IOExceptio
-        BitbucketBranchSource bitbucketBranchSource = multiBranchJob.addBranchSource(BitbucketBranchSource.class);
-        bitbucketBranchSource
-                .credentialsId(bbsAdminCredsId)
-                .serverId(serverId)
-                .projectName(forkRepo.getProject().getKey())
-                .repositoryName(forkRepo.getSlug());
-        multiBranchJob.save();
-
-        // Clone (fork) repo
-        File checkoutDir = tempFolder.newFolder(REPOSITORY_CHECKOUT_DIR_NAME);
-        Git gitRepo = cloneRepo(ADMIN_CREDENTIALS_PROVIDER, checkoutDir, forkRepo);
-
-        // Push new Jenkinsfile to master
-        RevCommit masterCommit =
-                commitAndPushFile(gitRepo, ADMIN_CREDENTIALS_PROVIDER, MASTER_BRANCH_NAME, checkoutDir,
-                        JENKINS_FILE_NAME, ECHO_ONLY_JENKINS_FILE_CONTENT.getBytes(UTF_8));
-        String masterCommitId = masterCommit.getId().getName();
-
-        // Push Jenkinsfile to feature branch
-        final String branchName = "feature/test-feature";
-        gitRepo.branchCreate().setName(branchName).call();
-        RevCommit featureBranchCommit =
-                commitAndPushFile(gitRepo, ADMIN_CREDENTIALS_PROVIDER, branchName, checkoutDir, JENKINS_FILE_NAME,
-                        ECHO_ONLY_JENKINS_FILE_CONTENT.getBytes(UTF_8));
-        String featureBranchCommitId = featureBranchCommit.getId().getName();
-
-        multiBranchJob.open();
-        multiBranchJob.reIndex();
-        multiBranchJob.waitForBranchIndexingFinished(30);
-
-        // Verify build was triggered
-        WorkflowJob masterJob = multiBranchJob.getJob(MASTER_BRANCH_NAME);
-        Build lastMasterBuild = masterJob.getLastBuild();
-        assertThat(lastMasterBuild.getResult(), is(SUCCESS.name()));
-
-        String encodedBranchName = URLEncoder.encode(branchName, UTF_8.name());
-        WorkflowJob featureBranchJob = multiBranchJob.getJob(encodedBranchName);
-        Build lastFeatureBranchBuild = featureBranchJob.getLastBuild();
-        assertThat(lastFeatureBranchBuild.getResult(), is(SUCCESS.name()));
-
-        // Verify BB has received build status
-        fetchBuildStatusesFromBitbucket(masterCommitId).forEach(status ->
-                assertThat(status,
-                        successfulBuildWithKey(multiBranchJob.name + "/" + lastMasterBuild.job.name)));
-
-        fetchBuildStatusesFromBitbucket(featureBranchCommitId).forEach(status ->
-                assertThat(status,
-                        successfulBuildWithKey(multiBranchJob.name + "/" + lastFeatureBranchBuild.job.name)));
+    public void testFullBuildFlowWithMultiBranchJobAndManualReIndexingWithBranchAndPrDiscovery()
+            throws IOException, GitAPIException {
+        runFullFlowMultiBranchWithManualReindexing(true, true);
     }
 
     @Test
-    public void testFullBuildFlowWithMultiBranchJobAndBitbucketWebhookTrigger() throws IOException, GitAPIException {
```

Comment:
I removed this test as it was very similar to the `runFullFlowMultiBranch()` tests but let me know if I'm missing something and need to add it back in.

Reply:
It is very similar, but they test different things.
The removed test tested that the manual scan (so a user clicking "Scan multibranch pipeline now") works and finds the relevant branch. The `runFullFlowMultiBranch` runs a build *but* relies on the triggers and tests that the triggers fire as they should. 

I see that we have a `runFullFlowMultiBranchWithManualReindexing` that is doing the same test as this one, so with that in mind I think it is fine to remove this particular test, and rely on the new `runFullFlowMultiBranchWithManualReindexing` instead.